src/datetimepicker.ios.js

```diff
@@ -62,11 +64,16 @@ export default function Picker({
   sharedPropsValidation({value});
 
   const [heightStyle, setHeightStyle] = React.useState(undefined);
-  const _picker: NativeRef = React.useRef(null);
+  const _picker = React.useRef(null);
   const display = getDisplaySafe(providedDisplay);
 
   React.useEffect(
     function ensureNativeIsInSyncWithJS() {
+      if (isFabricEnabled) {
+        // we don't need this workaround when fabric is enabled
```

Comment:
turns out it wasn't needed even before, tit was just a remainder from when this was refactored from a class component :D
so you can remove the effect